kafka-ui-react-app/src/components/Wizard/WizardForm/SchemaRegistry/SchemaRegistry.tsx

```diff
@@ -5,104 +5,117 @@ import Input from 'components/common/Input/Input';
 import { FormError } from 'components/common/Input/Input.styled';
 import { ErrorMessage } from '@hookform/error-message';
 import { useFormContext } from 'react-hook-form';
+import Heading from 'components/common/heading/Heading.styled';
+import { InputLabel } from 'components/common/Input/InputLabel.styled';
 
 const SchemaRegistry = () => {
-  const [newSchemaRegistry, setNewSchemaRegistry] = useState(false);
-  const methods = useFormContext();
-  const registry: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+  const [isShowRegistryForm, setIsShowRegistryForm] = useState(true);
+  const {
+    getValues,
+    reset,
+    register,
+    watch,
+    formState: { errors },
+  } = useFormContext();
+
+  const showRegistryFrom: React.MouseEventHandler<HTMLButtonElement> = (e) => {
     e.preventDefault();
-    setNewSchemaRegistry(!newSchemaRegistry);
-    if (!newSchemaRegistry) {
-      methods.reset({ url: '' });
+    setIsShowRegistryForm(!isShowRegistryForm);
+    if (!isShowRegistryForm) {
+      reset({
```

Comment:
setValue. We should not reset form

Reply:
Reset leads to updating of default value it is a wrong way

Reply:
we can add just {keepDefaultValue: true},
but i will do it with setValue